components/automate-ui/src/app/services/user-preferences/user-preferences.actions.ts

```diff
@@ -0,0 +1,64 @@
+import { HttpErrorResponse } from '@angular/common/http';
+import { Action } from '@ngrx/store';
+import {
+  UserPreferencesPayload, UserPreferenceResponse
+} from './user-preferences.model';
+
+export enum UserPreferencesActionTypes {
```

Comment:
Align this enum as we are aligning in different action files:
https://github.com/chef/automate/blob/b8486c44deeb30362f8691fe1fa5f5025d60e042/components/automate-ui/src/app/entities/infra-roles/infra-role.action.ts#L5-L21

Reply:
done